pkg/controller/watch.go

```diff
@@ -57,6 +56,17 @@ func (v *VarnishController) watchConfigUpdates(ctx context.Context, c *exec.Cmd,
 	}
 }
 
+func (v *VarnishController) setTemplate(tmplContents []byte) error {
+	parsedTemplate, err := template.New("vcl").Parse(string(tmplContents))
+	if err == nil {
+		v.vclTemplate = parsedTemplate
+		hash := md5.Sum(tmplContents)
+		hashStr := hex.EncodeToString(hash[:])
+		v.vclTemplateHash = hashStr
+	}
+	return err
```

Comment:
This is a matter of preference, but I think the usual `if err != nil`-approach might be more intuitive, here.

```suggestion
	parsedTemplate, err := template.New("vcl").Parse(string(tmplContents))
	if err != nil {
		return err
	}

	v.vclTemplate = parsedTemplate
	hash := md5.Sum(tmplContents)
	hashStr := hex.EncodeToString(hash[:])
	v.vclTemplateHash = hashStr

	return nil
```

Reply:
done